### ISSUE TITLE
missing SP5106_DSM_1M.asc

### ISSUE THREAD

**Issue by Krandelbord** [catenoid/Shadow-Map#1]
I was trying to run this code, but it fails on missing SP5106_DSM_1M.asc file. 
What kind of data should be inside of this file ?

**Comment by catenoid** [comment]
This file contained the Lidar data set, which was initially sourced from the Geomatics Groups in February 2014 under an academic license. (I copied wiki content to the readme to make it more visible, which contains sample images from the script.) In September 2015, Lidar for England became open data. In particular, this was the statement from the Environment Agency's Acorn newsletter:

"Users will be able to access our 2m, 1m and 50cm LIDAR composite data as DTM and DSM for free under an OGL without the need to register. The 25cm composite data and Tiled LIDAR (specific dated surveys) will be released in a phased approach in the following days and weeks. From mid September you will no longer be able to access LIDAR data via the Geomatics web portal."

So in short, the original source for SP5106_DSM_1M.asc has closed down, and under the initial license I cannot share this file. However you should now be able to acquire Lidar data free from the Environment Agency.
